Svelte guide not working due to NodeJS version

### Expected Behavior
The page will be deployed as stated in the guide: [Guide](https://developers.cloudflare.com/pages/framework-guides/deploy-a-svelte-site)

### Actual Behavior
The pipeline failes due to old NodeJS version: 
`npm ERR! notsup Unsupported engine for @sveltejs/kit@1.0.0-next.113: wanted: {"node":"^12.20 || ^14.13.1 || >= 16"} (current: {"node":"12.18.0","npm":"6.14.4"})`

### Section that requires update
Either the guide needs to be updated to show a solution on this conflict or the pipeline needs to be upgraded to a newer version of NodeJS

--- a/products/pages/src/content/framework-guides/deploy-a-svelte-site.md
+++ b/products/pages/src/content/framework-guides/deploy-a-svelte-site.md
@@ -55,8 +55,7 @@ Select the new GitHub repository that you created above and, in the configuratio
 | Production branch     | `main`             |
 | Build command         | `npm run build`    |
 | Build directory       | `build`            |
-| Environment Variables	| `NODE_VERSION: 14` |
-
+| Environment Variables | `NODE_VERSION: 14` |
 
 </TableLayout>
 

--- a/products/pages/src/content/framework-guides/deploy-a-svelte-site.md
+++ b/products/pages/src/content/framework-guides/deploy-a-svelte-site.md
@@ -50,15 +50,23 @@ Select the new GitHub repository that you created above and, in the configuratio
 
 <TableLayout>
 
-| Configuration option | Value           |
-| -------------------- | --------------- |
-| Production branch    | `main`          |
-| Build command        | `npm run build` |
-| Build directory      | `build`          |
+| Configuration option  | Value              |
+| --------------------- | ------------------ |
+| Production branch     | `main`             |
+| Build command         | `npm run build`    |
+| Build directory       | `build`            |
+| Environment Variables	| `NODE_VERSION: 14` |
+
 
 </TableLayout>
 
 Optionally, you can customize the `Project Name` setting. It defaults to the repository's name, but it does not need to match. This value is assigned as your `pages.dev` subdomain.
+
+<Aside type="warning">
+
+**Important:** SvelteKit requires Node.js v14.x or later to build successfully! You must expand the "Environment Variables (advanced)" section and add a `NODE_VERSION` variable with a value of `14` or greater.
+
+</Aside>
 
 ### SvelteKit Configuration
 

